plugins/SSH.koplugin/main.lua

```diff
@@ -29,11 +29,21 @@ local SSH = WidgetContainer:extend{
 function SSH:init()
     self.SSH_port = G_reader_settings:readSetting("SSH_port") or "2222"
     self.allow_no_password = G_reader_settings:isTrue("SSH_allow_no_password")
+    self.autostart = G_reader_settings:isTrue("SSH_autostart")
+
+    if self.autostart then
+        self:start()
+    end
+
     self.ui.menu:registerToMainMenu(self)
     self:onDispatcherRegisterActions()
 end
 
 function SSH:start()
+    if self:isRunning() then
+        return
```

Comment:
That's probably worth a debug or even an info log? In case it causes anything confusing.

Reply:
Sounds good, I did consider having it produce a slight variation on the "server started" but that would possibly be noisier than desired.